cmd/kube-proxy/app/server.go

```diff
@@ -369,6 +370,14 @@ func (o *Options) applyDeprecatedHealthzPortToConfig() {
 		return
 	}
 
+	// check ipv6
```

Comment:
While you are here, do you mind somehow refactoring this out and reuse it on `SetDefaults` as well?
https://github.com/kubernetes/kubernetes/blob/37e6266ef4b203a3241bf140f36ff8ae29d2e1f7/pkg/proxy/apis/config/v1alpha1/defaults.go#L36-L49

You change current only has impact on flags, which are deprecated.

Reply:
Good. I'm going to refactor this.
Thanks